xml/System.IO.Pipelines/PipeOptions.xml

```diff
@@ -45,14 +45,14 @@
         <Parameter Name="useSynchronizationContext" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="pool">To be added.</param>
-        <param name="readerScheduler">To be added.</param>
-        <param name="writerScheduler">To be added.</param>
-        <param name="pauseWriterThreshold">To be added.</param>
-        <param name="resumeWriterThreshold">To be added.</param>
-        <param name="minimumSegmentSize">To be added.</param>
-        <param name="useSynchronizationContext">To be added.</param>
-        <summary>Creates a new instance of <see cref="T:System.IO.Pipelines.PipeOptions" />.</summary>
+        <param name="pool">The pool of memory blocks to be used for buffer management.</param>
+        <param name="readerScheduler">The <see cref="T:System.IO.Pipelines.PipeScheduler" /> to be used to execute <see cref="T:System.IO.Pipelines.PipeReader" /> callbacks.</param>
+        <param name="writerScheduler">The <see cref="T:System.IO.Pipelines.PipeScheduler" /> used to execute <see cref="T:System.IO.Pipelines.PipeWriter" /> callbacks.</param>
+        <param name="pauseWriterThreshold">The number of bytes in the <see cref="T:System.IO.Pipelines.Pipe" /> when <see cref="M:System.IO.Pipelines.PipeWriter.FlushAsync(System.Threading.CancellationToken)" /> starts blocking.</param>
```

Comment:
I like "before" instead of "when" for these. It's also probably worth pointing out that setting these to a negative value makes this unlimited.

```suggestion
        <param name="pauseWriterThreshold">The number of bytes in the <see cref="T:System.IO.Pipelines.Pipe" /> before <see cref="M:System.IO.Pipelines.PipeWriter.FlushAsync(System.Threading.CancellationToken)" /> starts blocking. A negative value prevents <see cref="M:System.IO.Pipelines.PipeWriter.FlushAsync(System.Threading.CancellationToken)" /> from ever blocking effectively making the number of bytes in the <see cref="T:System.IO.Pipelines.Pipe" /> unlimited.</param>
```

Reply:
I like your suggestion, thanks. I also added a comma.